src/components/Dialogs/ActionsModal.tsx

```diff
@@ -93,21 +117,14 @@ function ActionsModal() {
     }
   };
 
-  const renderItem = (item: { name: string; function: string; data?:boolean; }) => (
-    <div className="flex justify-between font-semibold">
-      <span>{item.name}</span>
-      <span className="text-highlight-1" onClick={() => runAction(item.name, item.function)}><Icon className="cursor-pointer" path={mdiPlayCircleOutline} size={1} /></span>
-    </div>
-  );
-
   const renderTab = (title, items) => (
     <React.Fragment>
       <div className="grow px-4 py-2 bg-background-alt self-stretch border-b border-background-border shadow flex justify-between">
         <p className="text-base font-semibold text-gray-300">{title}</p>
         <span onClick={() => { setActiveTab(title); }}><Icon className="cursor-pointer" path={mdiChevronUp} size={1} rotate={activeTab === title ? 0 : 180} /></span>
       </div>
       <div className={cx('flex flex-col grow w-full p-4 space-y-1', { hidden: activeTab !== title })}>
-        {items.map(item => renderItem(quickActions[item]))}
+        {items.map(item => <RenderItem {...quickActions[item]} runAction={runAction} />)}
```

Comment:
If we can avoid it then please don't use the spread operator when rendering react elements. Also, if we don't spread then we don't need to pass the unused `data` property (until it's used at least) amd can omitn8t from the props object used by the `RenderItem` function.